docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.1'
+
+services:
+  your_bot_service_name:
+    image: your_bot_image_name
+    container_name: your_bot_container_name
```

Comment:
https://docs.docker.com/compose/environment-variables/

it would be better if you read this and rewrite compose file to use .env with ```docker-compose config``` command